README.md

```diff
@@ -1 +1,35 @@
-# Multi-Compartment-Segmentation
\ No newline at end of file
+# Multi Compartment Segmentation
+
+Multi compartment segmentation is a [detectron2](https://github.com/facebookresearch/detectron2) based panoptic segmentation model that is implemented as a pipeline in [Digital Slide Archive](https://athena.rc.ufl.edu/) (DSA) developed and maintained by [CMI lab](https://cmilab.nephrology.medicine.ufl.edu/) at the University of Florida. This DSA plugin allows users to segment a kidney Whole Slide Image (WSI) into 6 compartments, namely: Interstitium,medulla, glomerulus, sclerotic glomerulus, tubule, and arterial tree.
+
+## Quick Start Guide
+
+This DSA plugin allows users to run image analysis jobs through the HistomicsUI by the following steps:
+
+1. Create an account at [DSA](https://athena.rc.ufl.edu/) and log in with your credentials. Please contact Sayat Mimar at sayat.mimar@ufl.edu for obtaining a DSA account.
```

Comment:
@sayatmimar: Why people need to contact Sayat Mimar for creating a DSA account? If you want to know your affiliations, you can tell them to put their affiliations when they register.